LightBulb.WindowsApi/PowerBroadcast.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Windows.Forms;
+using LightBulb.WindowsApi.Internal;
+
+
+namespace LightBulb.WindowsApi
+{
+    public partial class PowerBroadcast
+    {
+        public Action<byte> Handler { get; }
```

Comment:
Do we need the `byte` part? Are we doing anything with it? If not, we can drop it.